spec/factories/checkins.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :checkin do
+    
+  end
+end
```

Comment:
It's always better to *remove* code that you don't know if you need, especially when you're creating new code. Otherwise you run the risk of adding more complexity that you have to wade through to figure out what is going on and what is absolutely necessary.